### ISSUE TITLE
adding google assistent as a part of nvda.

### ISSUE THREAD

**Issue by alireza1385mamani** [nvaccess/nvda#13847]
### Is your feature request related to a problem? Please describe.
hello and thank you for creating this very good screen reader. as a blind user who uses voices instead of keyboard in moast cases, 
for me it is a real problem that this screenreader doesn't have a voice assistent built in to it.
### Describe the solution you'd like
I'd like to see a voice assistent like google com in to the nvda features
### Describe alternatives you've considered
n/a
### Additional context
you can use the google assistent api and use google as youre default assistent. it is much easier than building youre owne voice assistent. you can read more about google assistent's api and sdk at [heare](https://developers.google.com/assistant/sdk)

**Comment by TheQuinbox** [comment]
This is a feature better suited for an NVDA add-on, IMO. JAWS added one of these, and it did nothing but added bloat and a feature that a lot of their users didn't use. For the users that use it that's great, they should be able to install it, not have it built into your screen reader.

**Comment by alireza1385mamani** [comment]
You have no other options can you use Siri API or like Amazon Alexa API if
you can tell me how I didn't find any information regarding this

On Mon, Jun 27, 2022, 9:40 PM Brian Gaff ***@***.***> wrote:
>
> I notice after a big fanfare a couple of years ago how interfaces could
be
> built that mimic conversations, for things like booking rooms etc, it now
> seems that conversational actions are going away from the realms of third
> party developers by next year
> Google has a very bad track record of inventing ideas then just
abandoning
> them for no apparent reason, so do people really want to build their
> systems around using such an API?
> I doubt it.
> Brian
>
> --
> ***@***.***
> Sent via blueyonder.(Virgin media)
> Please address personal E-mail to:-
> ***@***.***, putting 'Brian Gaff'
> in the display name field.
> ----- Original Message -----
> From: "alireza1385mamani" ***@***.***>
> To: "nvaccess/nvda" ***@***.***>
> Cc: "Subscribed" ***@***.***>
> Sent: Monday, June 27, 2022 8:52 AM
> Subject: [nvaccess/nvda] adding google assistent as a part of nvda.
(Issue
> #13847)
>
>
> > 
> > hello and thank you for creating this very good screen reader. as a
blind
> > user who uses voices instead of keyboard in moast cases, I'd like to
see a
> > voice assistent like google com in to the nvda featurs.
> > ### Is your feature request related to a problem? Please describe.
> > hello and thank you for creating this very good screen reader. as a
blind
> > user who uses voices instead of keyboard in moast cases,
> > for me it is a real problem that this screenreader doesn't have a voice
> > assistent built in to it.
> > ### Describe the solution you'd like
> > I'd like to see a voice assistent like google com in to the nvda
features
> > ### Describe alternatives you've considered
> > n/a
> > ### Additional context
> > you can use the google assistent api and use google as youre default
> > assistent. it is much easier than building youre owne voice assistent.
you
> > can read more about google assistent's api and sdk at
> > [heare](https://developers.google.com/assistant/sdk)
> >
> > --
> > Reply to this email directly or view it on GitHub:
> > https://github.com/nvaccess/nvda/issues/13847
> > You are receiving this because you are subscribed to this thread.
> >
> > Message ID: ***@***.***>
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.Message ID:
***@***.***>

**Comment by alireza1385mamani** [comment]
Yes definitely it's a good idea

**Comment by mwhapples** [comment]
Then there is cortana in Windows. Whilst nobody ever seems to talk about cortana, may be with it being in windows it may offer greater intergration possibilities.
The point though being there are many voice assistants, everyone may have their preference. Building it into NVDA is bad as it offers the user no choice as to which they use. If it is done as an addon then the user only needs to install the one they want, if they want any at all.

**Comment by alireza1385mamani** [comment]
Well I have the same not really good at assisting it's not as good as
Google assistant and and I think it's better to include an add-on 4 either
Google assistant Amazon Alexa or Apple Siri they are more popular

**Comment by seanbudd** [comment]
Closing in favour of #12489
